### PR TITLE
chore(dashmate): rename high-performance nodes to evo nodes

### PR DESCRIPTION
**Pull request by shuplenkov**
## Issue being fixed or feature implemented

Rename High-performance nodes into Evo nodes

## What was done?

High-performance nodes were renamed into Evo nodes

## How Has This Been Tested?

Manual

## Breaking Changes

No

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/dashmate/src/listr/tasks/setup/setupRegularPresetTaskFactory.js [hunk 1]
```diff
@@ -62,10 +62,10 @@ function setupRegularPresetTaskFactory(
...
     Masternode                  - Full node features, plus Core services such as
                                   ChainLocks and InstantSend (1000 DASH collateral)
-    High-performance full node  - Full node features, plus host a full copy of the
-                                  Platform blockchain (no collateral)
-    High-performance masternode - Masternode features, plus Platform services such
-                                  as DAPI and Drive (4000 DASH collateral)\n`,
+    Evo full node  - Full node features, plus host a full copy of the Platform
+                                  blockchain (no collateral)
+    Evo masternode - Masternode features, plus Platform services such as DAPI and
+                                  Drive (4000 DASH collateral)\n`,
```
Comment (thephez) [inline packages/dashmate/src/listr/tasks/setup/setupRegularPresetTaskFactory.js:68]: Maybe we should spell out "Evolution" here? "Evo" is kinda an abbreviation.

Comment (shumkov) [reply packages/dashmate/src/listr/tasks/setup/setupRegularPresetTaskFactory.js:68]: Sounds good to me

File: packages/dashmate/src/listr/tasks/setup/nodeTypes.js [hunk 2]
```diff
@@ -5,9 +5,9 @@ const {
 
 const NODE_TYPE_NAMES = {
   MASTERNODE: 'masternode',
-  HP_MASTERNODE: 'high-performance masternode',
+  HP_MASTERNODE: 'evo masternode',
```
Comment (shumkov) [inline packages/dashmate/src/listr/tasks/setup/nodeTypes.js:8]: evolution

**Review by shumkov — APPROVED** [review]
👍

**Review by thephez — APPROVED** [review]

**Pull request merged**